mlc/index.py

```diff
@@ -116,10 +174,62 @@ def build_index(self):
                     # Check for configuration files (meta.yaml or meta.json)
                     for config_file in ["meta.yaml", "meta.json"]:
                         config_path = os.path.join(automation_path, config_file)
-                        if os.path.isfile(config_path):
+                        if os.path.isfile(config_path):   
+                            key = f"{repo_path}/{folder_type}/{automation_dir}"
+                            current_script_keys.add(key)
+                            mtime = self.get_script_mtime(automation_path)
```

Comment:
this is executed for all targets like script, cache and experiment right? So, `get_item_mtime` may be a better name